besu/src/test/java/org/hyperledger/besu/ForkIdsNetworkConfigTest.java

```diff
@@ -84,10 +84,9 @@ public static Collection<Object[]> parameters() {
           new Object[] {
             NetworkName.HOLESKY,
             List.of(
-                new ForkId(Bytes.ofUnsignedInt(0xa200f558L), 1694790240L),
-                new ForkId(Bytes.ofUnsignedInt(0x840a3b53L), 2000000000L),
-                new ForkId(Bytes.ofUnsignedInt(0x30771f90), 0L),
-                new ForkId(Bytes.ofUnsignedInt(0x30771f90), 0L))
+                new ForkId(Bytes.ofUnsignedInt(0xc61a6098L), 1696000704L),
```

Comment:
Couldn't find a reference for this, would be good to cross reference with other clients once they've made the changes